Package.swift

```diff
@@ -26,9 +26,14 @@ let package = Package(
       url: "https://github.com/apple/swift-argument-parser.git",
       .upToNextMinor(from: "0.3.0")
     ),
+    .package(
+      name: "SwiftPM",
+      url: "https://github.com/apple/swift-package-manager.git",
+      .branch("main")
+    ),
     .package(
       url: "https://github.com/apple/swift-tools-support-core.git",
-      .upToNextMinor(from: "0.1.10")
+      .branch("main")
```

Comment:
SwiftPM repo has tags like `swift-release-5.3.1` as far as I remember. Maybe we could rely on those?

Reply:
Disregard that, I tried with `swift-5.3.2-RELEASE` and it doesn't work because it tries to pull the `master` branch of `swift-llbuild` which doesn't exist anymore... 😕

Reply:
Yes, saw that. Also the 5.3.x tags seem especially out of sync vs the `release/5.3` branches. I've opened a small [PR](https://github.com/apple/swift-package-manager/pull/3145) for the llbuild dependency.